manifests/cs-dialogue/zh-en.jsonl

```diff
@@ -0,0 +1,1028 @@
+{"dataset_id": "cs-dialogue", "sample_id": "ZH-CN_U0017_S0_2", "src_audio": "cs-dialogue/short_wav/WAVE/C0/ZH-CN_U0017_S0/ZH-CN_U0017_S0_2.wav", "src_ref": "Hello hello，很高兴认识你啊。", "tgt_ref": null, "src_lang": "zh", "tgt_lang": "en", "benchmark_metadata": {"cs_lang": ["en", "zh"], "context": "short", "dataset_type": "code_switch"}}
```

Comment:
Please store the audios in `cs-dialogue/audio/zh` as it is the structure we chose to have for all datasets